console/src/main/webapp/manager/app/app.es6

```diff
@@ -73,18 +73,43 @@ class AppController {
 }
 
 class StandaloneController {
-  static $inject = ['$scope', 'Orgs', 'User']
+  static $inject = ['$injector', '$window', '$scope', 'Orgs', 'User']
 
-  constructor ($scope, Org) {
-    if (!window.org) {
+  constructor ($injector, $window, $scope, Org) {
+    if (!$window.org) {
       $scope.org = new Org()
       return
     }
 
-    $scope.org = window.org
-    $scope.users = window.org.members
-    $scope.isReferentOrSuperUser = window.isReferentOrSuperUser
-    $scope.gdprAllowAccountDeletion = window.gdprAllowAccountDeletion
+    this.$injector = $injector
```

Comment:
doesn't looks necessary to store it in object.

Reply:
right